src/web/index.css

```
@@ -4066,3 +4066,162 @@ textarea {
+  border-top: 1px solid var(--color-border-light);
+  display: flex;
+  flex-direction: column;
+  gap: 8px;
+  animation: filterBarSlideDown 0.22s cubic-bezier(0.22, 1, 0.36, 1);
+}
+
+.route-filter-bar-expanded.is-closing {
+  animation: filterBarSlideUp 0.18s ease forwards;
+}
+
+@keyframes filterBarSlideDown {
```

Stylelint (17.4.0)

[error] Expected keyframe name "filterBarSlideDown" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)